src/components/project/MediaDetailsComp.vue

```diff
@@ -135,7 +146,7 @@ function getHitsMessage(mediaObj) {
                 @close="showZoomModal = false"
               >
                 <MediaViewPanel
-                  :imgSrc="buildImageProps(media_file.media['original'])"
+                  :imgSrc="getMediaUrl('large')"
```

Comment:
In this part, it's likely we will need to use the original. But we need to reimplement a more complicated media viewer anyway.